pom.xml

```diff
@@ -1905,11 +1905,28 @@
                     <groupId>com.github.eirslett</groupId>
                     <artifactId>frontend-maven-plugin</artifactId>
                     <version>${frontend.plugin.version}</version>
+                    <executions>
+                        <execution>
+                            <goals>
+                                <goal>install-node-and-pnpm</goal>
+                            </goals>
+                        </execution>
+                        <execution>
+                            <id>pnpm-install</id>
+                            <goals>
+                                <goal>pnpm</goal>
+                            </goals>
+                            <configuration>
+                                <arguments>${pnpm.args.install}</arguments>
+                                <workingDirectory>${maven.multiModuleProjectDirectory}</workingDirectory>
+                            </configuration>
+                        </execution>
+                    </executions>
```

Comment:
This is what we had originally I believe, the POM in the `js` directory would run the install only once. The problem with that is if you build a child module (e.g. `js/apps/admin-ui`), it would not install the dependencies causing the build to fail, unless you build the parent first.

Not sure if this suggestion addresses that, but let's try to take care of this in another PR/issue, as I don't want to scope creep this specific PR.